libs/tup-components/src/accounts/ManageAccount.tsx

```diff
@@ -7,62 +7,60 @@ import styles from './ManageAccount.module.css';
 const ManageUser = () => (
   <>
     <div className={styles['tap-header']}>
-      <strong>Edit User Information</strong>
+      <strong>Account Managment</strong>
     </div>
-    <span className={styles['tap-description']}>
-      Your user account can be managed on the TACC Account Management (TAM)
-      portal.
-    </span>
+    <p className={styles['tap-description']}>
+      Account details are mananged by the TACC Account Management portal. Follow
+      the links below to:
+    </p>
     <a
       href="https://accounts.tacc.utexas.edu/profile"
       target="_blank"
       rel="noreferrer"
       className={styles['tap-href']}
     >
-      <Button className={styles['tap-button']} type="primary">
+      <Button className={styles['tap-button']} type="secondary">
         Edit User Profile
       </Button>
     </a>
-  </>
-);
-
-const ManageDNs = () => (
```

Comment:
**Beware the position of items in diff is misleading.** The `manageDNs` is shown further down. The `managePassword` is removed.